src/image.rs

```diff
@@ -147,3 +154,180 @@ impl Image {
         self
     }
 }
+
+/// Describes the image content of a filled or stroked shape.
+///
+/// See also [`ImageBrushRef`] which can be used to avoid allocations.
+#[derive(Clone, PartialEq, Debug)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct ImageBrush {
```

Comment:
I think Chad's comment:

> Nothing else in peniko uses a `Brush` suffix so I think we want to avoid that here.

has been unaddressed, as far as I'm aware.
Personally, I'm of two minds here; I think that the reasons to not call this just `Image` stand, but I think `SampledImage` isn't quite right, nor is `ImageDraw` (which really is the same as `Brush`)

Reply:
I'm happy enough there.